packages/mask/src/plugins/Avatar/hooks/useNFT.ts

```diff
@@ -35,7 +36,13 @@ export function useNFT(
         const amount = asset?.priceInToken?.amount ?? asset?.price?.[CurrencyType.USD] ?? '0'
         const name = contract?.name || metadata?.name || ''
         const imageURL = metadata?.imageURL
-        const permalink = asset?.link ?? Others?.explorerResolver.nonFungibleTokenLink(chainId, address, tokenId)
+        let permalink = asset?.link
+        if (!permalink) {
+            permalink =
+                pluginId === NetworkPluginID.PLUGIN_SOLANA
+                    ? MagicEdenAPI.getUrl(tokenId)
+                    : Others?.explorerResolver.nonFungibleTokenLink(chainId, address, tokenId)
+        }
```

Comment:
```suggestion
        const permalink = asset?.link ?? (pluginId === NetworkPluginID.PLUGIN_SOLANA
                    ? MagicEdenAPI.getUrl(tokenId)
                    : Others?.explorerResolver.nonFungibleTokenLink(chainId, address, tokenId))
```

Reply:
Updated.

However, I prefer simple expression which easy to understand in a glance.

Reply:
You cannot easily tell when the `let` variable will change. So it's better to avoid using it.